chore: small storybook updates (#643)
ing-bank/lion: packages/form-system/stories/17-Validation-Examples.stories.mdx

></lion-input>
```

## Default messages

To get default validation messages you need to import and call the `loadDefaultFeedbackMessages` function once in your application.

> Sometimes the default messages doesn't make sense for your input field. In that case you want to override it by adding a `getMessage` function to your validator.

Reviewer: `messages don't make sense`
```suggestion
Sometimes the default messages don't make sense for your input field. In that case you want to override it by adding a `getMessage` function to your validator.
```

## Override fieldName

> In other cases the default messages are correct, but you only want to change the `fieldName`. This can be done for only one specific validator or for all at once.

Reviewer: ```suggestion
In the scenario that the default messages are correct, but you only want to change the `fieldName`, this can both be done for a single specific validator or for all at once.
```
